.gitignore

```diff
@@ -27,6 +27,8 @@ id_rsa*
 hosts*
 *.flag
 keys.txt
+results/
```

Comment:
Added a new script in the IO500 example that will scp the IO500 tar.gz results files into a results directory.  Adding that to .gitignore.